pkg/validation/options.go

```diff
@@ -28,11 +30,22 @@ import (
 // are of the correct format
 func Validate(o *options.Options) error {
 	if o.SSLInsecureSkipVerify {
-		// TODO: Accept a certificate bundle.
 		insecureTransport := &http.Transport{
 			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 		}
 		http.DefaultClient = &http.Client{Transport: insecureTransport}
+	} else if o.ProviderCAFiles != nil && len(o.ProviderCAFiles) > 0 {
```

Comment:
Is `ProviderCAFiles` a pointer? I didn't think it was? We shouldn't need the `!= nil` check here I don't think

Reply:
It is not a pointer, `!=` removed.